app/Http/Controllers/RestApi/RestApiMember.php

```diff
@@ -126,7 +126,7 @@ private function getAllowedRequestedGroups(Request $request, $group_ids)
             $requestedGroups = [];
             
             /** @var GroupRepository $groupRepository */
-            $groupRepository = ApiHelper::createGroupRepo($request->header($key = 'db_key'));
+            $groupRepository = ApiHelper::createGroupRepo();
```

Comment:
Können wir das mit dem Default Key so belassen oder braucht es für die gecacheten Gruppen mehr Sicherheit in Zukunft?

Reply:
Grundsätzlich gibt es ja immer noch einen Rechte-Check auf den einzelnen Gruppen. Wenn jemand also nur auf einen Teilbaum zugreiffen kann, sieht er auch nur diese Gruppen. Jedoch sieht er über den Root-Path der Gruppe, wie weit er vom Root weg ist und welche IDs der Root sowie die Gruppen dazwischen haben. Diese Information würde ich aber nicht als besonders sensitiv klassifizieren (zumal man auch dazu authentifiziert sein muss). Von daher scheint es mir kein Problem.